### PR TITLE
hip: Adding hipother for cuda support starting from 6.0

### PR DESCRIPTION
**Pull request by renjithravindrankannath**

### CODE REVIEW

File: var/spack/repos/builtin/packages/hip/package.py [hunk 1]
```diff
@@ -552,8 +569,12 @@ def cmake_args(self):
             )
             args.append(self.define("HIP_RUNTIME", "rocclr"))
             args.append(self.define("HIP_PLATFORM", "amd"))
+            if "@5.6.0:" in self.spec:
```
Comment (srekolam) [inline var/spack/repos/builtin/packages/hip/package.py]: if self.spec.satisfies("@5.6.0:"):

**Review by tldahlgren — APPROVED** [review]
LGTM

**Comment by tldahlgren** [comment]
Will wait to merge to see if @srekolam and or @haampie want to comment.

**Comment by tldahlgren** [comment]
> Will wait to merge to see if @srekolam and or @haampie want to comment.

OR I can merge "now" if no-one objects.  Let me know.

**Review by srekolam — APPROVED** [review]

**Pull request merged**